testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/federation/storage/UserStorageTest.java

```diff
@@ -102,6 +104,7 @@
  *
  * @author tkyjovsk
  */
+@EnableFeature(Profile.Feature.LEGACY_COOKIES)
```

Comment:
Since the legacy cookies are going to be removed, could you instead update the `testLoginSuccessWithSpecialCharacter` not to check legacy cookie?

Reply:
I'd rather keep it like this and remove both the profile and test code at a later point in time. Even thought we're deprecating this profile, we should still test this functionality while the profile exists.